components/project-card/project-card.jsx

```diff
@@ -217,11 +300,11 @@ class ProjectCard extends React.Component {
   }
 
   renderDescription() {
-    return this.props.description.split(`\n`).map(paragraph => <p>{paragraph}</p>);
+    return this.props.description.split(`\n`).map((paragraph, i) => <p key={i}>{paragraph}</p>);
```

Comment:
Oh yea I thought about that but wasn't sure if having a long key (`paragraph`) would be problematic. Will update this line.